packages/deployment/src/init.js

```diff
@@ -262,13 +265,21 @@ const askProvider = ({ inquirer }) => PROVIDERS => {
   return inquirer.prompt(questions);
 };
 
-const doInit = ({ env, rd, wr, running, setup, inquirer, fetch }) => async (
-  progname,
-  args,
-) => {
+const doInit = ({
+  env,
+  rd,
+  wr,
+  running,
+  setup,
+  inquirer,
+  fetch,
+  parseArgs,
```

Comment:
`parseArgs` is powerful? the way it's used here doesn't seem to be.

Reply:
Yes, you're probably right that I overcompensated by passing in `parseArgs` when I should have just imported `minimist` in the `init.js`.

Can't win. 😉